pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+authors = [
+    {name = "Tom Hu", email = "thomas.hu@sentry.io"},
+    {name = "Codecov Support", email = "support@codecov.io"},
+]
+description = "Codecov Command Line Interface"
+requires-python = ">= 3.9"
+dependencies = [
+    "click==8.*",
+    "httpx==0.27.*",
+    "ijson==3.*",
+    "pyyaml==6.*",
+    "regex",
+    "responses==0.21.*",
+    "sentry-sdk>=2.20.0",
+    "test-results-parser==0.5.*",
+    "tree-sitter==0.20.*",
+    "wrapt>=1.17.2",
+]
+license = {file = "LICENSE"}
+name = "codecov-cli"
+readme = "README.md"
+version = "10.0.1"
+
+[project.scripts]
+codecovcli = "codecov_cli.main:run"
+
+[tool.setuptools]
+packages = ["codecov_cli"]
```

Comment:
I haven't tested this but give removing a try. Modern setuptools tend to auto-detect such stuff:
```suggestion
```

I recommend building both sdist and wheel before and after, unpacking them (`tar xzvf` for sdist and `wheel unpack`/`unzip` for the wheel), and comparing with a recursive diff. Just to confirm that the difference is acceptable. If it picks up the package dir, then you can drop it from the config.

Reply:
Ahhh, the reason for this was an error running `pypa/build`

```
 Multiple top-level packages discovered in a flat-layout: ['hooks', 'samples', 'languages', 'wheelhouse', 'codecov_cli'].
```